src/main/scala/org/nbastats/model/HasBasicStats.scala

```diff
@@ -1,10 +1,24 @@
 package org.nbastats.model
 
+import java.sql.Time
+
 trait HasBasicStats {
 
-  val games: Int
-  val pointsPerGame: Double
-  val reboundsPerGame: Double
-  val assistsPerGame: Double
+  val game: Int
+  val fga: Int
+  val fgm: Int
+  val threesA: Int
+  val threesM: Int
+  val fta: Int
+  val ftm: Int
+  val points: Int
+  val rebounds: Int
+  val assists: Int
+  val blocks: Int
+  val steals: Int
+  val turnovers: Int
+  val timePlayed: Time
```

Comment:
i think we should use a `Duration` for this

Reply:
changes have been made